packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -279,6 +283,16 @@ export default class NewVm extends BaseComponent {
         share: this._getResourceSet()?.shareByDefault ?? false,
       })
     }
+
+    const pool = this.props.pool
+    if (get(() => prevProps.pool.id) !== get(() => pool.id)) {
+      const master = this.props.hosts[pool.master]
+      this.setState({
+        poolGuestSecurebootReadiness: semver.satisfies(master?.version, '>= 8.3.0')
+          ? await getPoolGuestSecureBootReadiness(pool.id)
+          : undefined,
+      })
+    }
```

Comment:
If the VM is being created with Self Service, `pool` is `undefined`. In that case, we'd need to check the template's pool.

Reply:
It depends if knowing the secure boot readiness of the pool is sensitive information or not. /cc @stormi 
It only tells you if the *VM* is going to be able to use Secure Boot, right? And it seems important for a VM user to get that information even if they don't have permissions on the pool... I think? :thinking:

Reply:
I don't think it's sensitive information.